wiki/en/Running-a-Server.md

```diff
@@ -98,7 +98,7 @@ In this mode your Server will appear in the server list supplied by a Directory.
 
 Alternatively, you can list your Server on a Custom Directory (see below).  Clients will only find your Server if they enter the Custom Directory's internet address.
 
-Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server becuase the Directory is full, you can try registering on another. 
+Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server because the Directory is full, you can try registering on a different Directory. 
```

Comment:
I'm still not comfortable with "registering _on_ a Directory" as a phrase.  "With" feels better.  You don't "go to a party on a friend", you "go to a party with a friend"... ("go to a party" = "register"; "a friend" = "a Directory").
```suggestion
Note that Directories can only register up to 150 Servers. If you see a message that says you cannot register your Server because the Directory is full, you can try registering with a different Directory. 
```

Reply:
OK will also add that to the style guide if I can remember.